src/devices/Mcp3428/ConversionResult.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Mcp3428
+{
+    public readonly struct ConversionResult
+    {
+        public ConversionResult(byte channel, short rawData, ResolutionEnum resolution)
+        {
+            Channel = channel;
+            RawData = rawData;
+            Resolution = resolution;
+        }
+
+        /// <summary>
+        /// Gets the channel.
+        /// </summary>
+        /// <value>The channel.</value>
+        /// <autogeneratedoc />
+        public byte Channel { get; }
+
+        /// <summary>
+        /// Gets the raw data.
+        /// </summary>
+        /// <value>The raw data.</value>
+        /// <autogeneratedoc />
+        public Int16 RawData { get; }
```

Comment:
nit: `Int16` => `short`

Reply:
This one had more meaning for me. Will read more about why is this recommended.